src/librbd/exclusive_lock/AcquireRequest.h

```diff
@@ -14,15 +14,21 @@ class Context;
 
 namespace librbd {
 
+namespace managed_lock{
+class LockWatcher;
+}
+
 template <typename> class Journal;
+template <typename> class Lock;
+typedef Lock<librbd::managed_lock::LockWatcher> LockT;
```

Comment:
Minor: move this into the class instead of polluting the librbd namespace. Of course, might not be needed anymore, either.